tests/test_rule_handling.py

```diff
@@ -113,7 +116,46 @@ def storage_mock():
     return mock.Mock()
 
 
-@mock.patch("pyanaconda.dbus.DBus.get_proxy")
+@pytest.fixture()
+def proxy_getter(monkeypatch):
```

Comment:
Could you please clarify what is going on here? I think that you have replaced the need for a mock injection on a per-test case basis to injecting a mock factory at the top, but I am not sure how exactly it works.
Fixtures with arguments are a less-known feature of `pytest`, so we have to figure out how to describe this, so developers are not confused and they can continue writing tests in the same spirit.